api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -59,7 +56,7 @@
   private final AccessTierService accessTierService;
   private final BillingProjectBufferEntryDao billingProjectBufferEntryDao;
   private final Clock clock;
-  private final FireCloudService fireCloudService;
+  public final FireCloudService fireCloudService;
```

Comment:
Why?

Reply:
Even if this was done for testing, please find a different way. This should not be necessary

Reply:
intellij did that. Reverted.